Rename monitoring token (#118)
ObolNetwork/obol-docs: docs/int/quickstart/group/quickstart-group-operator.md

This process can take a minimum of 16 hours, with the maximum time to activation being dictated by the length of the activation queue, which can be weeks. You can leave your distributed validator cluster offline until closer to the activation period if you would prefer. You can also use this time to improve and harden your monitoring and alerting for the cluster.

## Step 6 - Optional. Add the Monitoring Credentials

:::info

> This step is **optional** but will help the Obol Team monitor the health of your cluster. It can only be perfomed if the Obol Team has contacted you directly.

Reviewer: ```suggestion
This step is **optional** but will help the Obol Team monitor the health of your cluster. It can only be perfomed if the Obol Team has given you a credential to use.
```

Lots of teams say "we will never contact you", to make the fake sale/airdrop messages less believable. We still will probs DM them to people, but we shouldn't prime people to expect an outreach from us.
